### PR TITLE
Add August Bluetooth as a supported brand of Yale Access Bluetooth

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Add August Bluetooth as a supported brand of Yale Access Bluetooth

Assa Abloy, Yale's parent company, purchased August in 2017, and most newer devices use the Yale Access branding.

Although Assa Abloy has not released anything new under the August brand since 2020, they still sell under the August brand in at least North America. I expect people will likely have trouble finding August Bluetooth support under Yale Access Bluetooth since they seem to have actively tried not to mix the branding.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/76625
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3583
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/august_ble.markdown [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: August Bluetooth
+description: Instructions on how to integrate August Bluetooth devices into Home Assistant.
```
Comment (bdraco) [inline source/_integrations/august_ble.markdown]:
```suggestion
description: Instructions on how to integrate August Bluetooth locks into Home Assistant.
```

File: source/_integrations/august_ble.markdown [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ha_supporting_integration: Yale Access Bluetooth
+---
+
+Integrates [August](https://august.com/) locks over Bluetooth into Home Assistant. To integrate with the August cloud service, or non-lock devices, see the [August](/integrations/august) page.
+
+Following Assa Abloy, Yale's parent company, purchasing August in 2017, most newer devices use the Yale Access branding.
+
+{% include integrations/config_flow.md domain=page.ha_supporting_domain %}
+<br><br>
+To find more information, please use the [Yale Access Bluetooth](/integrations/yalexs_ble) page.
```
Comment (frenck) [inline source/_integrations/august_ble.markdown]: This is a supported brand, we have templating for that.

```suggestion
Integrates [August](https://august.com/) locks over Bluetooth into Home Assistant.

Following Assa Abloy, Yale's parent company, purchasing August in 2017, most newer devices use the Yale Access branding.

{% include integrations/supported_brand.md %}
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/august_ble.markdown (hunk 2 above)
Comment (bdraco) [reply source/_integrations/august_ble.markdown]: Oh nice.  I copied the original from the Leviton zwave one

**Review by frenck — APPROVED** [review]
Thanks, @bdraco 👍

**Comment by bdraco** [comment]
Thanks

**Pull request merged**